release-team/role-handbooks/release-notes/README.md

```diff
@@ -68,15 +68,13 @@ Update this section at the end of each release for the next team
 
 ### Week 9
 
-- Former Release Notes lead helps identify new leads and shadows
 - Create Google Doc for generated release notes and share with release-notes team
 - Share created doc with release-team
 - Start determining major themes for release notes template to send to sig-leads
 
 
 ### Week 10
 
-- Finalize lead and shadow roles for subsequent release
```

Comment:
I think this is still valid.

Reply:
ok I will rebase with this change. Thanks!